components/hooks/useDatawrapper.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+
+export default function useDatawrapper(csv, indicator, location, chartType) {
+  const [chartId, setChartId] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    fetch("/api/datawrapper-proxy", {
+      method: "POST",
+      body: JSON.stringify({
+        csv,
+        indicator,
+        location,
+        chartType,
+      }),
+    })
+      .then((resolve) => resolve.json())
```

Comment:
Any chance of some syntactically sugary async await here? Or does the useEffect bar that option?

Reply:
I tried it, but eslint wasn't too happy (orange underlines so allowable but not proper I think) not quite sure why

Reply:
Interesting!